skl2onnx/_parse.py

```diff
@@ -160,6 +160,18 @@ def _parse_sklearn_feature_union(scope, model, inputs, custom_parsers=None):
             _parse_sklearn_simple_model(
                 scope, transform, inputs,
                 custom_parsers=custom_parsers)[0])
+        if model.transformer_weights is not None:
+            transform_result = [transformed_result_names.pop()]
+            # Create a Multiply ONNX node
+            multiply_operator = scope.declare_local_operator('SklearnMultiply')
+            multiply_operator.inputs = transform_result
+            assert name in model.transformer_weights
```

Comment:
An explicit message would be better or remove it if it is always true.

Reply:
Actually scikit just keeps the actual value in this case, I'll change the behaviour to align with it.